promql/promqltest/testdata/native_histograms.test

```diff
@@ -998,6 +998,26 @@ eval instant at 10m histogram_sum(increase(reset_in_bucket[15m]))
 
 clear
 
+# Test that abs() silently ignores native histograms
+load 5m
+    multi_histogram	{{schema:0 sum:5 count:4 buckets:[1 2 1]}}x10
+
+eval instant at 10m abs(histogram_count(multi_histogram))
+    {} 4
+
+eval range from 9m to 10m step 1m abs(histogram_count(multi_histogram))
```

Comment:
@beorn7  might be overkill to add range version as well, especially  since I'll add `ceil` later, wdyt?

Reply:
I don't think we are doing it for all the other test cases. I think it's reasonable to assume that we can rely on the general semantics that a range query is equivalent to a number of instant queries. (Of course, the implementation should be tested for fulfilling this assumption, but that's a separate thing that we do not need to prove again for every single PromQL function.)

BTW: IIRC, the testing framework runs every `instant` test as a `range` test with just one sample under the hood.